plugins/linguist-backend/src/db/LinguistBackendDatabase.ts

```diff
@@ -102,13 +104,13 @@ export class LinguistBackendDatabase implements LinguistBackendStore {
     }
 
     try {
-      return JSON.parse(entityResults.languages);
+      return JSON.parse(entityResults.languages as string);
```

Comment:
is this cast really needed inside JSON.parse?

Reply:
Yeah, this can be refactored. Thanks!